lib/Test/Nginx/Util.pm

```diff
@@ -2398,13 +2398,13 @@ END {
 sub can_run {
     my ($cmd) = @_;
 
-    #warn "can run: @_\n";
-    #my $_cmd = $cmd;
-    #return $_cmd if (-x $_cmd or $_cmd = MM->maybe_command($_cmd));
+    #warn "can run: $cmd\n";
+    return $cmd if -f $cmd && -x $cmd;
```

Comment:
Just pushed an updated patch, is that what you meant? Additionally, are we fine with calling `get_nginx_version()` before `can_run()` like currently?